test/unit/util_spec.js

```diff
@@ -50,6 +50,90 @@ describe('util', function() {
     });
   });
 
+  describe('isArrayBuffer', function() {
+    it('handles array buffer values', function() {
+      expect(isArrayBuffer(new ArrayBuffer(0))).toEqual(true);
+    });
+
+    it('handles non-array buffer values', function() {
+      expect(isArrayBuffer('true')).toEqual(false);
+      expect(isArrayBuffer(1)).toEqual(false);
+      expect(isArrayBuffer(null)).toEqual(false);
+      expect(isArrayBuffer(undefined)).toEqual(false);
+    });
+  });
+
+  describe('isBool', function() {
+    it('handles boolean values', function() {
+      expect(isBool(true)).toEqual(true);
+      expect(isBool(false)).toEqual(true);
+    });
+
+    it('handles non-boolean values', function() {
+      expect(isBool('true')).toEqual(false);
+      expect(isBool('false')).toEqual(false);
+      expect(isBool(1)).toEqual(false);
+      expect(isBool(0)).toEqual(false);
+      expect(isBool(null)).toEqual(false);
+      expect(isBool(undefined)).toEqual(false);
+    });
+  });
+
+  describe('isEmptyObj', function() {
+    it('handles empty objects', function() {
+      expect(isEmptyObj({})).toEqual(true);
+    });
+
+    it('handles non-empty objects', function() {
+      expect(isEmptyObj({ foo: 'bar', })).toEqual(false);
+    });
+  });
+
+  describe('isNum', function() {
+    it('handles numeric values', function() {
```

Comment:
Please add an additional test-case with a floating point value, to test that `isNum` works in that case as well (to ensure that `isNum` is different from the native `Number.isInteger` function).